pkg/operator/api/userconfig/config.go

```diff
@@ -493,5 +494,9 @@ func ReadAppName(filePath string, relativePath string) (string, error) {
 		return "", errors.Wrap(ErrorMissingAppDefinition(), relativePath)
 	}
 
+	if err := urls.CheckDNS1123(appName); err != nil {
```

Comment:
Is this necessary since it should be caught by configreader? Can you try removing the check here and seeing if it still catches it?